server/helper/mailSender.js

```diff
@@ -0,0 +1,60 @@
+var MailConnection = require('../mailConnection');
+
+//TIP: Nodemailer is unicode friendly ✔
+//receiverList is an array of strings, which must as follows:
+//  "[NAME SURNAME] email"
+function mailSender (receiversList, subject, text, html){
+
+    var errorMessage = '';
+    var mailOptions={};
+    var transporter = MailConnection.transporter;
+    var user= MailConnection.user;
+
+
+    //check we have at least one destination mail, and put the first into mailOptions
+    if(receiversList.length > 0){
+        mailOptions.to= receiversList[0];
+    }
+    else {
+        return errorMessage = 'Destination mail not provided';
+
+    }
+
+    //get the rest of destinations if any
+    for (var i=1; i<receiversList.length; i++){
```

Comment:
Use forEach method for iterating the array:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach